cmd/bnsd/x/account/codec.proto

```diff
@@ -31,6 +31,13 @@ message Domain {
   // Account review defines the duration of the account renewal period for each
   // account that belongs to this domain.
   int64 account_renew = 7 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixDuration"];
+  // Broker is an arbitrary data that can be provided by a middleman
+  // (i.e. retailer) that although does not participate directly in the
+  // transaction, helped a user to register a domain. Storing a third party
+  // token helps identify contribution of such party later. Must be an email or
+  // a weave address that starts with a format or hex.
+  // example bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws
```

Comment:
Broker is a weave address (bech32 or hex) that can be provided by a middleman that helped facilitate the registration transaction.  For example, an IOV token holder that registers a domain in exchange for fiat from a client is a broker.  Storing the broker helps identify the contribution of such a party, which allows for automated commission distribution through an IOV reward initiative, for example.  Must be a weave address that starts with a format or hex, for example: bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws.